fdbserver/DataDistribution.actor.cpp

```diff
@@ -1167,14 +1167,15 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 			.detail("MachineMaxTeams", maxMachineTeams);
 	}
 
-	int overlappingMembers( vector<UID> &team ) {
+	int overlappingMembers(const vector<UID>& team) const {
 		if (team.empty()) {
 			return 0;
 		}
 
 		int maxMatchingServers = 0;
-		UID& serverID = team[0];
-		for (auto& usedTeam : server_info[serverID]->teams) {
+		const UID& serverID = team[0];
+		const auto& usedTeams = server_info.find(serverID)->second->teams;
```

Comment:
`overlappingMembers` should only be called if the team exists in the `server_info` map, so this should not cause a behaviour change, I think `operator[]` was originally used just because it's quicker to write. I will add an assertion for this invariant.